text/0003-measure-metric-type.md

```diff
@@ -2,50 +2,169 @@
 
-## Internal details
+The preceding raw statistics API supported all-or-none recording for interdependent measurements.  This RFC introduces a `RecordBatch` API to support recording batches of measurements in a single API call, where a `Measurement` is now defined as a tuple of `MeasureMetric`, `Value` (integer or floating point), and `Labels`.
 
-The type known as `MeasureMetric` is a direct replacement for the raw statistics `Measure` type.  The `MeasureMetric.Record` method records a single observation of the metric.  The `MeasureMetric.GetOrCreateTimeSeries` supports pre-defined labels, just the same as `Gauge` and `Cumulative` metrics.
+# Explanation
 
-## Trade-offs and mitigations
+The common use for `MeasureMetric`, like the preceding raw statistics API, is for reporting information about rates and distributions over structured, numerical event data.  Measure metrics are the most general-purpose of metrics.  Informally, the individual metric event has a logical format expressed as one primary key=value (the metric name and a numerical value) and any number of secondary key=values (the labels, resources, and context).
 
-This Measure Metric API is conceptually close to the Prometheus [Histogram, Summary, and Untyped metric types](https://prometheus.io/docs/concepts/metric_types/), but there is no way in OpenTelemetry to distinguish these cases at the declaration site, in code.  This topic is covered in 0004-metric-configurable-aggregation.
+    metric_name=_number_
+    pre_defined1=_any_value_
+    pre_defined2=_any_value_
+    ...
+    resource1=_any_value_
+    resource2=_any_value_
+    ...
+    context_tag1=_any_value_
+    context_tag2=_any_value_
+    ...
+
+Here, "pre_defined" keys are those captured in the metrics handle, "resource" keys are those configured when the SDK was initialized, and "context_tag" keys are those propagated via context.
+
+Events of this form can logically capture a single update to a named metric, whether a cumulative, gauge, or measure kind of metric.  This logical structure defines a _low-level encoding_ of any metric event, across the three kinds of metric.  An SDK could simply encode a stream of these events and the consumer, provided access to the metric definition, should be able to interpret these events according to the semantics prescribed for each kind of metric.
+
+## Metrics API concepts
+
+The `Meter` interface represents the metrics portion of the OpenTelemetry API.
+
+There are three kinds of metric instrument, `CumulativeMetric`, `GaugeMetric`, and `MeasureMetric`.
+
+Metric instruments are constructed by the API, they are not constructed by any specific SDK.
+
+| Name | A string. |
```

Comment:
The formatting of this table looks broken to me: https://github.com/open-telemetry/oteps/blob/a8b7baf3d8a0514a70b770543e010e4dff6260d7/text/0003-measure-metric-type.md

Reply:
@jmacd please fix